app/src/main/java/com/concordium/wallet/ui/account/accountsoverview/AccountsOverviewViewModel.kt

```diff
@@ -204,6 +208,22 @@ class AccountsOverviewViewModel(application: Application) : AndroidViewModel(app
         }
     }
 
+    private suspend fun handleAccountCreation(notifyWaitingLiveData: Boolean) {
+        val allAccounts = accountRepository.getAll()
+        if (allAccounts.isNotEmpty() && allAccounts.any { it.transactionStatus == TransactionStatus.FINALIZED }) {
+            App.appCore.session.hasCompletedOnboarding()
+            postState(OnboardingState.DONE, notifyWaitingLiveData = notifyWaitingLiveData)
+            showSingleDialogIfNeeded()
+        } else {
+            postState(
+                state = OnboardingState.FINALIZING_ACCOUNT,
+                notifyWaitingLiveData = notifyWaitingLiveData
+            )
+            viewModelScope.launch { restartUpdater(5L) }
```

Comment:
Please, add this value to the build.gradle as well, can be called `FAST_ACCOUNT_UPDATE_FREQUENCY_SEC`